### PR TITLE
Rename "default repos" to "indexable repos"

### PR DESCRIPTION
**Pull request by ryanslade**
Default repos doesn't really convey the intent anymore. It was
originally just listing repos from the default_repos table on
sourcegraph.com so that they would be indexed. We are now pulling in
repos from more places so calling it "default repos" is confusing. The
intent is still that any repo included should be indexed.

### CODE REVIEW

File: enterprise/internal/insights/discovery/all_repos_iterator_test.go [hunk 1]
```diff
@@ -196,11 +196,11 @@ func TestAllReposIterator_DotCom(t *testing.T) {
 		autogold.Want("items0", []string{"1", "2", "3", "4", "5", "6", "7", "8", "9"}).Equal(t, each)
 	}
 
-	// Were the DefaultRepoStore.List calls as we expected?
-	autogold.Want("defaultRepoStoreListCalls0", int(1)).Equal(t, defaultRepoStoreListCalls)
+	// Were the IndexableRepoStore.List calls as we expected?
+	autogold.Want("defaultRepoStoreListCalls0", int(1)).Equal(t, indexableRepoStoreListCalls)
```
Comment (ryanslade) [inline enterprise/internal/insights/discovery/all_repos_iterator_test.go:200]: @slimsag I wasn't sure how to change `defaultRepoStoreListCalls0` here

File: enterprise/cmd/executor/internal/worker/mock_command_runner_test.go [hunk 2]
```diff
@@ -4,9 +4,8 @@ package worker
 
 import (
 	"context"
-	"sync"
-
 	command "github.com/sourcegraph/sourcegraph/enterprise/cmd/executor/internal/command"
+	"sync"
```
Comment (efritz) [inline enterprise/cmd/executor/internal/worker/mock_command_runner_test.go:8]: These seem very wrong.

**Review by efritz — CHANGES_REQUESTED** [review]
I think there's a lot of unrelated import changes.

**Review by unknwon — APPROVED** [review]
Didn't look into code as I expect just bunch of renaming, but the idea SGTM!

File: enterprise/cmd/executor/internal/worker/mock_command_runner_test.go (hunk 2 above)
Comment (ryanslade) [reply enterprise/cmd/executor/internal/worker/mock_command_runner_test.go:8]: Ah, that's goimports trying to organise the imports and failing.. Not sure why it's not grouping stdlib imports together as it should

**Comment by rvantonder** [comment]
@keegancsmith agree-I've been exploring defining queries for each kind of search, and "Default DotCom repositories" is one that definitely deserves it own designation. I've been using `DotComDefault` as a name for a `repo` query that queries this set.

**Comment by ryanslade** [comment]
Closing this as there wasn't enough consensus at to whether this was an improvement :)

**Pull request closed without merging**